gnat2goto/driver/tree_walk.adb

```diff
@@ -839,6 +842,26 @@ package body Tree_Walk is
       return Args;
    end Do_Call_Parameters;
 
+   ------------------------
+   -- Do_If_Expression --
+   ------------------------
+
+   function Do_If_Expression (N : Node_Id) return Irep is
+      Expr : Node_Id := First (Expressions (N));
+      Cond : constant Irep := Do_Expression (Expr);
+      First : Irep;
+      Second : Irep;
+      Expr_Type : constant Irep := Do_Type_Reference (Etype (N));
+   begin
```

Comment:
What's "First" and "Second" here? They are apparently subexpressions of the if expression, but which ones?

Reply:
Yeah, I'll rename them to something more descriptive.